test/c/derive.c

```diff
@@ -192,6 +192,78 @@ void derive_single_line_with_custom_knots(CuTest *tc)
 	TS_END_TRY
 }
 
+void derive_single_line_with_custom_knots_degree_2(CuTest *tc)
+{
+	tsBSpline spline = ts_bspline_init();
+	tsDeBoorNet net = ts_deboornet_init();
+	tsReal *result = NULL;
+	tsStatus status;
+	tsReal ctrlp[3];
+	tsReal knots[6];
+
+    size_t degree = 2;
```

Comment:
Indention

Reply:
Good catch. Addressed in https://github.com/msteinbeck/tinyspline/pull/163/commits/35327e87dd8b7c22084dc3054af11877a7ee836b